src/lib/PickerComponent.ios.js

```diff
@@ -25,7 +25,6 @@ class RenderedSelector extends React.Component {
   render() {
     let picker = (
       <Picker
-        testID="Picker"
```

Comment:
fyi removing this might affect some existing tests - used in `AppForm` and `InspectionChecklistPage`